apps/dashboard/app/helpers/projects_helper.rb

```diff
@@ -14,4 +14,53 @@ def render_readme(readme_location)
       simple_format(file_content)
     end
   end
+
+  # Organizes job information in a manner that is easily displayed
+  def readable_job_data(job)
+    account_info = {}
+    status_info = {}
+    job_info = {}
+    job.to_h.each do |key, value|
+      unless value.nil?
+        case key
+        when 'job_name'
+          account_info['Job Name'] = value
```

Comment:
With just 1 table, this should simplify a lot. But in any case, I'm not in love with the giant case statement. Maybe try a lookup table to map the key `job_name` to the new key `Job Name`. You may even get by with a simple `map` to `key.humanize`. 

Also I'd say put this somewhere as a member of the `HpcJob` class so we can just do `job.to_human_display` or something similar.